Lib/posixpath.py

```diff
@@ -135,6 +135,35 @@ def splitdrive(p):
     return p[:0], p
 
 
+def splitroot(p):
+    """Split a pathname into drive, root and tail. On Posix, drive is always
+    empty; the root may be empty, a single slash, or two slashes. The tail
+    contains anything after the root. For example:
+
+        splitdrive('foo/bar') == ('', '', 'foo/bar')
+        splitdrive('/foo/bar') == ('', '/', 'foo/bar')
```

Comment:
```suggestion
        splitroot('foo/bar') == ('', '', 'foo/bar')
        splitroot('/foo/bar') == ('', '/', 'foo/bar')
```

I'd also add an example with the two-slashes case